src/main/scala/com/snowplowanalytics/kinesistee/KinesisTee.scala

```diff
@@ -86,6 +86,7 @@ object KinesisTee extends Tee {
     content
       .map(transform)
       .filter(filter)
+      .grouped(100)
```

Comment:
Couldn't we have this parameter as a configuration?

Reply:
I have update PR and made this configurable.